hack/.linted_packages

```diff
@@ -26,6 +26,7 @@ cmd/kubectl
 cmd/kubelet
 cmd/libs/go2idl/client-gen
 cmd/libs/go2idl/client-gen/generators
+cmd/libs/go2idl/client-gen/path
```

Comment:
Eh, this is the only change to hack/ and I don't think it is worth bothering one of hack/OWNERS to approve it. I'll try to manually add the approve tag.